src/renderer/components/snipping-tool.tsx

```diff
@@ -270,12 +278,16 @@ const SnippingTool: React.FunctionComponent<ISnippingToolProps> = ({
       : 'MERGE_FAIL';
     sendAnalyticsToMain(
       AnalyticsElements.SCREEN_CAPTURE_ANNOTATE,
-      ScreenSnippetActionTypes.ANNOTATE_DONE,
```

Comment:
Hi @NguyenTranHoangSym could you please revert this change? ANNOTATE_ADD --> ANNOTATE_DONE

Reply:
done